apps/dashboard/app/(app)/logs/components/log-details/components/meta-content.tsx

```diff
@@ -0,0 +1,37 @@
+import { Card, CardContent } from "@/components/ui/card";
+import { useEffect, useState } from "react";
+import { createHighlighter } from "shiki";
+
+const highlighter = createHighlighter({
+  themes: ["github-light", "github-dark"],
+  langs: ["json"],
+});
+
+export function MetaContent({ content }: { content: any }) {
+  const [innerHtml, setHtml] = useState("Loading...");
+
+  useEffect(() => {
+    highlighter.then((highlight) => {
+      const html = highlight.codeToHtml(JSON.stringify(content), {
+        lang: "json",
+        themes: {
+          dark: "github-dark",
+          light: "github-light",
+        },
+        mergeWhitespaces: true,
+      });
+      setHtml(html);
+    });
+  }, [content]);
+
+  return (
+    <Card className="rounded-[5px]">
+      <CardContent
+        className="whitespace-pre-wrap text-[12px] w-[300px]"
+        dangerouslySetInnerHTML={{
```

Comment:
why do we need to use this?

Reply:
Code formatted by "shiki" doesn't work if not injected like this. But, i'll double check anyways. We might have to purify them just incase.

Reply:
why do we need shiki anyway?
what's wrong with a good old `pre` tag?

Reply:
Yeah, highlighting doesn't have any benefits for our meta fields. Initially I was using this for every jsonish field for log details.